lib/registrant.rb

```diff
@@ -0,0 +1,19 @@
+class Registrant
+  attr_reader :name, :age, :permit, :license_data
```

Comment:
Careful here, we didn't want a method called `.permit` so we don't want it in the attr_reader.